modelci/types/bo/dynamic_profile_result_bo.py

```diff
@@ -160,23 +160,29 @@ def __init__(self,
     def to_dynamic_profile_result_po(self):
         """Convert business object to plain object for persistence.
         """
-        dpr_po = DynamicProfileResultPO(ip=str(self.ip), device_id=self.device_id, device_name=self.device_name,
-                                        batch=self.batch, memory=self.memory.memory, cpu_memory=self.memory.cpu_memory,
-                                        gpu_memory=self.memory.gpu_memory,
-                                        initialization_latency=self.latency.init_latency.tolist(),
-                                        preprocess_latency=self.latency.preprocess_latency.tolist(),
-                                        inference_latency=self.latency.inference_latency.tolist(),
-                                        postprocess_latency=self.latency.postprocess_latency.tolist(),
-                                        batch_formation_throughput=self.throughput.batch_formation_throughput.tolist(),
-                                        preprocess_throughput=self.throughput.preprocess_throughput.tolist(),
-                                        inference_throughput=self.throughput.inference_throughput.tolist(),
-                                        postprocess_throughput=self.throughput.postprocess_throughput.tolist(),
-                                        create_time=self.create_time)
+        dpr_po = DynamicProfileResultDO(
```

Comment:
Resolve #81 first so we can update the profiler's features.

Reply:
agree

Check issue first

Reply:
will do after this merge